### PR TITLE
add ProgressIndicator

### LINKED ISSUE #124: ProgressIndicator
As mentioned in #96 this is a seperate issue for the ProgressIndicator

### PR DESCRIPTION
**Pull request by DasProffi**
closes #124
Unbenannt

### CODE REVIEW

**Comment by MaxSchaefer** [comment]
> @MaxSchaefer do you have any idea why our ci fails?

`#17 0.854  ERR_PNPM_FROZEN_LOCKFILE_WITH_OUTDATED_LOCKFILE  Cannot perform a frozen installation because the lockfile needs updates`

The `pnpm-lock.yaml` is out-of-sync.

https://github.com/pnpm/pnpm/issues/2807#issuecomment-682471525

**Comment by DasProffi** [comment]
> [pnpm/pnpm#2807 (comment)](https://github.com/pnpm/pnpm/issues/2807#issuecomment-682471525)
Doesn't work, `pnpm install` doesn't change the lock file, I tried this even with completly deleting the local repository.

Could someone else verify that the lock file isn't updated and if it does, push the commit? @MaxSchaefer

**Comment by MaxSchaefer** [comment]
Of course, we should also use a current PNPM version in the container. Why it has worked for us in CI so far, good question. Since PNPM 8 came out today, and we were on version 6, I assume that this has contributed to the fail of the CI.

File: tasks/components/ProgressIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { tw } from '@helpwave/common/twind/index'
+
+export type ProgressIndicatorProps = {
+  progress: number,
+  strokeWidth?: number,
+  size?: 'small' | 'medium' | 'big',
```
Comment (MaxSchaefer) [inline tasks/components/ProgressIndicator.tsx:6]:
```suggestion
  size?: keyof typeof sizeMapping,
```

File: tasks/components/ProgressIndicator.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { tw } from '@helpwave/common/twind/index'
+
+export type ProgressIndicatorProps = {
+  progress: number,
```
Comment (MaxSchaefer) [inline tasks/components/ProgressIndicator.tsx:4]: Please add some comments of the expected values. Like 0 to 1 for progress.

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]

**Comment by MaxSchaefer** [comment]
Nice job, I like the SVG approach. 👍

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**